style.css

```diff
@@ -1 +1,296 @@
-/*  Add your styling here */
+body {
+  box-sizing: border-box;
+}
+.navigation {
+  display: none;
+}
+.homepage-main {
+    column-gap: 10px;
+    row-gap: 10px;
+    display: grid;
+    /* grid-template-rows: repeat (5, auto); */
+    /*  grid-template-areas must follow the exact number of columns and rows defined above (4 columns and 4 rows)  */
+    grid-template-areas:
+      "hero"  
+      "welcome"         /* one for each column */
+      "feature"
+      "product"
+      "footer";           /* the dot is used to skip a column */
+  }
+  
+  .homepage-hero {
+    grid-area: hero;
+    display: grid;
+    background-size: cover;
+    background-image: url(/HTML-CSS-Coursework-Week3/assets/hero.jpeg);
+  }
+  .one-one {
+  justify-self: end;
+  padding: 15px;
+  }
+
+  .one-one div {
+  width: 35px;
+  height: 5px;
+  background-color: black;
+  margin: 6px 0;
+  }
+
+  .one-two {
+  justify-self: end;
+  align-self: flex-end;
+  font-style: italic;
+  font-size: 1.5rem;
+  padding-right: 10px;
+  padding-bottom: 10px;
+  }
+
+  .homepage-welcome {
+    grid-area: welcome;
+    padding: 15px;
+  }
+  
+  .homepage-feature {
+    grid-area: feature;
+    background-size: cover;
+  }
+
+  .homepage-feature img {
+    width: 100%;
+    object-fit: cover;
+  }
+  
+  .homepage-products {
+    grid-area: product;
+    gap: 0.5rem;
+    display: grid;
+    grid-template-areas:
+      "product-item1     product-item2"           /* one for each column */
+      "product-item3     product-item4"
+; 
+  }
+  .homepage-products img {
+    height: 8rem;
+    object-fit: cover;
+  }
+  .grid-item1 {
+    grid-area: product-item1;
+    justify-self: center;
+  }
+  .grid-item2 {
+    grid-area: product-item2;
+    justify-self: center;
+  }
+
+  .grid-item3 {
+    grid-area: product-item3;
+    justify-self: center;
+  }
+
+  .grid-item4 {
+    grid-area: product-item4;
+    justify-self: center;
+  }
+  
+  .homepage-footer {
+    background-color:rgb(194, 229, 241);
+    grid-area: footer;
+    padding-left: 15px;
+  }
+  .oleh {
+    text-align: center;
```

Comment:
if you use the previous suggestions:
`grid-column-start: 2;` will center

Reply:
super, it helped to center it in all modes
<img width="710" alt="image" src="https://user-images.githubusercontent.com/12444326/201143967-d09541b2-55d2-4fc8-8d1f-34ddc47de65e.png">

Reply:
awesome. But the image you attached, still seems not totally centered. Probably is because of the class name called `contact` instead of `contacts`. Also in the design they seem to be all in the center compared to your image
<img width="930" alt="Screenshot 2022-11-10 at 17 03 28" src="https://user-images.githubusercontent.com/40305104/201145453-735a76fd-5f67-4b74-8e80-f630a14e4e4f.png">

Reply:
yes, you are right 
about wrong class name for contacts. now it is working fine

Reply:
<img width="744" alt="image" src="https://user-images.githubusercontent.com/12444326/201147430-f679b21e-d732-4b43-a851-dfc87d4fad42.png">